magma/primitives/memory.py

```diff
@@ -71,7 +70,8 @@ def _simulate(self, value_store, state_store):
 
 class Memory(Generator2):
     def __init__(self, height, T: Kind,
-                 read_latency: int = 0, read_only: bool = False):
+                 read_latency: int = 0, read_only: bool = False,
+                 init: Optional[tuple] = None):
```

Comment:
I never remember what the functional difference between `x: tuple` and `x: Tuple` is, but just making sure this is ok

Reply:
Hmm, they're really just cosmetic in this context so I don't think it matters, but Tuple is the actually Type (from typing) and tuple is the conversion operator (e.g. tuple(list(foo))), so if we were using something like mypy I think we would want Tuple